src/aura/GE_GiftEntryForm/GE_GiftEntryFormController.js

```diff
@@ -46,9 +46,7 @@
                         header: payload.modalProperties.header || '',
                         showCloseButton: payload.modalProperties.showCloseButton || true,
                         cssClass: component.getName() + ' custom-modal ' + payload.modalProperties.cssClass,
-                        closeCallback: payload.modalProperties.closeCallback || function () {
-                            component.set('v.isLoading', false);
-                        },
+                        closeCallback: payload.modalProperties.closeCallback,
```

Comment:
@psadie-sfdo I removed the anonymous function that was attempting to set isLoading back to false as it was causing an exception every time the review donations modal was closed in debug mode and isLoading was always undefined at that point anyway.  Let me know if you see any issues with this change!

Reply:
This change looks good to me! Thank you!

Reply:
Great thanks Prince!